builtins/amp-video.js

```diff
@@ -36,11 +36,31 @@ export function installVideo(win) {
     }
 
     /** @override */
-    layoutCallback() {
+    buildCallback() {
+      /** @private @const {?HTMLVideoElement} */
+      this.video_ = document.createElement('video');
       const width = this.element.getAttribute('width');
       const height = this.element.getAttribute('height');
-      const video = document.createElement('video');
-      if (!video.play) {
+
+      this.video_.width = getLengthNumeral(width);
+      this.video_.height = getLengthNumeral(height);
+
+      const posterAttr = this.element.getAttribute('poster');
+      if (!posterAttr && getMode().development) {
+        console/*OK*/.error(
+            'No "poster" attribute has been provided for amp-video.');
+      }
+
+      // Disable video preload in prerender mode.
+      this.video_.setAttribute('preload', 'none');
+      this.propagateAttributes(['poster'], this.video_);
+      this.applyFillContent(this.video_, true);
+      this.element.appendChild(this.video_);
+    }
+
+    /** @override */
+    layoutCallback() {
+      if (!this.video_.play) {
```

Comment:
This is (my) the previously existing code. Expectation is that if `createElement('video')` does not have `play` method it means that the video is not supported on this platform.